### ISSUE TITLE
Get Nodemon to work

### ISSUE THREAD

**Issue by MelindaH26** [MelindaH26/Quote-API#1]
Get [Nodemon](https://www.npmjs.com/package/nodemon) to work in the dev environment so that the server doesn't have to be restarted every time a change is made to the code.

**Comment by nikhil3113** [comment]
hello @MelindaH26 i have solution, can i make a PR?

**Comment by Shoetan** [comment]
since you have nodemon installed as a dependency all you need to do is go to your package json, look for the scripts  and do two things. 
1. Modify the start command to 
 "scripts": {
    "start": "nodemon server.js"
  }
  
  2. Or write a new script 
   "scripts": {
    "start": "node server.js",
    "dev": "nodemon server.js"
  }

**Comment by MelindaH26** [comment]
@nikhil3113 Yes, of course! 

@Shoetan Thank you! I am not overly familiar with json scripts yet. Does your second example mean that the server could either be started with start ``npm run start`` or ``npm run dev``?

**Comment by Shoetan** [comment]
Yes you are right. You can have both scripts. Just know if you use node server.js any  change to your code you will have to kill and restart your server. If you use nodemon server.js there's no need for that.

**Comment by MelindaH26** [comment]
Great, thank you for the tips @Shoetan!

**Comment by Shoetan** [comment]
You're welcome @MelindaH26
